docs/reference/getting-started.asciidoc

```diff
@@ -1,740 +1,472 @@
+      "@timestamp": {
+        "gte": "2099-05-05",
+        "lt": "2099-05-08"
+      }
     }
-  }
+  },
+  "fields": [
+    "@timestamp"
+  ],
+  "_source": false,
+  "sort": [
+    {
+      "@timestamp": "desc"
+    }
+  ]
 }
---------------------------------------------------
+----
 // TEST[continued]
 
-Each `must`, `should`, and `must_not` element in a Boolean query is referred
-to as a query clause. How well a document meets the criteria in each `must` or
-`should` clause contributes to the document's _relevance score_. The higher the
-score, the better the document matches your search criteria. By default, {es}
-returns documents ranked by these relevance scores. 
-
-The criteria in a `must_not` clause is treated as a _filter_. It affects whether
-or not the document is included in the results, but does not contribute to
-how documents are scored. You can also explicitly specify arbitrary filters to
-include or exclude documents based on structured data. 
-
-For example, the following request uses a range filter to limit the results to
-accounts with a balance between $20,000 and $30,000 (inclusive). 
+Use <<date-math,date math>> to define relative time ranges. The following
+search will match not any documents in `logs-my_app-default`.
 
 [source,console]
---------------------------------------------------
-GET /bank/_search
+----
+GET logs-my_app-default/_search
 {
   "query": {
-    "bool": {
-      "must": { "match_all": {} },
-      "filter": {
-        "range": {
-          "balance": {
-            "gte": 20000,
-            "lte": 30000
-          }
-        }
+    "range": {
+      "@timestamp": {
+        "gte": "now-1d/d",
+        "lt": "now/d"
       }
     }
-  }
+  },
+  "fields": [
+    "@timestamp"
+  ],
+  "_source": false,
+  "sort": [
+    {
+      "@timestamp": "desc"
+    }
+  ]
 }
---------------------------------------------------
+----
 // TEST[continued]
 
-[[getting-started-aggregations]]
-== Analyze results with aggregations
+[discrete]
+[[extract-fields]]
+=== Step 8. Extract fields from unstructured content
 
-{es} aggregations enable you to get meta-information about your search results
-and answer questions like, "How many account holders are in Texas?" or 
-"What's the average balance of accounts in Tennessee?" You can search 
-documents, filter hits, and use aggregations to analyze the results all in one
-request. 
+Use a <<runtime-search-request,runtime field>> containing a
+<<modules-scripting-painless,Painless script>> and <<grok-basics,grok patterns>>
+to extract fields from unstructured content, such as log messages. Include the
+extracted field names in the `fields` parameter.
 
-For example, the following request uses a `terms` aggregation to group
-all of the accounts in the `bank` index by state, and returns the ten states
-with the most accounts in descending order:
+The following search uses the `runtime_mappings` parameter to extract the
+`source.ip` address from the `event.original` field.
```

Comment:
Ugh. I'm still kinda figuring out how to not write about runtime fields awkwardly. I kinda want to avoid introducing the concept of mappings here. I'll tinker with this and update. Thanks for calling this out!